src/errors/messages.ts

```diff
@@ -0,0 +1,91 @@
+const _messages = {
```

Comment:
Why not use an Enum here?

Reply:
Because an `enum` isn't appriopriate here.